gcp_variant_transforms/testing/integration/vcf_to_bq_tests/presubmit_tests/small_tests/incompatible_field_value.json

```diff
@@ -3,7 +3,8 @@
   "table_name": "incompatible_field_value",
   "input_pattern": "gs://gcp-variant-transforms-testfiles/small_tests/incompatible-field-value.vcf",
   "allow_incompatible_records": "True",
-  "runner": "DataflowRunner",
+  "runner": "DirectRunner",
+  "zones": "us-west1-b",
```

Comment:
Does this actually work? I thought we had to change this line: https://github.com/googlegenomics/gcp-variant-transforms/blob/master/gcp_variant_transforms/testing/integration/run_tests_common.py#L116

Reply:
Sorry I missed here. Updated now. PTAL :)